lua/nvim-treesitter/parsers.lua

```diff
@@ -2044,10 +2044,11 @@ list.slang = {
 
 list.slim = {
   install_info = {
-    url = "https://gitlab.com/theoreichel/tree-sitter-slim",
+    url = "https://github.com/theoo/tree-sitter-slim",
     files = { "src/parser.c", "src/scanner.c" },
   },
-  maintainers = { "@theoo", "@theoreichel" },
+  maintainers = { "@theoo" },
+  experimental = true,
```

Comment:
Sorry, but we don't accept experimental parsers anymore. If the grammar is not suitable for daily use (because the syntax is not fully covered or breaking changes are expected), then it's better to wait.

If it is, then it's not experimental ;)